### ISSUE TITLE
Disk Swap isnt working anymore

### ISSUE THREAD

**Issue by GoogleCodeExporter** [zabujard/nintendon-t#67]
```
What steps will reproduce the problem?
1.Get to disk swap screen by using disk 2 save if using disk 1 or starting new 
game on disk 2
2.Wait for Nintendont to try to read game disk
3.It never finishes

What is the expected output? What do you see instead?
Nintendont swapping disks correctly. It locks up during swap.  


What revision of Nintendont are you using? On what system Wii/Wii U?
v2.145 on Wii 

Please provide any additional information below.
I have checked Dolphin and it works fine. I think either Revision 135 aka 
(https://code.google.com/p/nintendon-t/source/detail?r=135) or 139 aka 
(https://code.google.com/p/nintendon-t/source/detail?r=139) caused it. It 
happens in at least Baten Kaitos Origins and Tales of Symphonia, and please 
report any other game it happens in. I didn't remember it happening in 1.126

```

Original issue reported on code.google.com by `gamemast...@gmail.com` on 21 Aug 2014 at 2:32

**Comment by GoogleCodeExporter** [comment]
```
Please try to figure out in which rev exactly it broke.
```

Original comment by `barcaped...@gmail.com` on 23 Aug 2014 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
Have you tried verifying the save files?
User ShadowOne333 and I think sonictopfan found out that using multi-MC 
emulation can cause that issue.
Try using individual saves for the 2 disc games and see if it works.
```

Original comment by `j.pedroz...@gmail.com` on 25 Aug 2014 at 6:24

**Comment by GoogleCodeExporter** [comment]
```
It never finishes even with MCEmu off and I use individual saves for both games 
sorry but that advice didn't work and it affects both games
```

Original comment by `gamemast...@gmail.com` on 26 Aug 2014 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
barcaped, it broke in 1.135 because 1.133 works fine
```

Original comment by `gamemast...@gmail.com` on 26 Aug 2014 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
Mark as Fixed fix94 fixed in 2.151
```

Original comment by `gamemast...@gmail.com` on 30 Aug 2014 at 7:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `Howard_M...@yahoo.com` on 30 Aug 2014 at 9:46
- Changed state: **Verified**
